Common/cpp/LayoutAnimations/LayoutAnimationsManager.cpp

```diff
@@ -16,13 +16,21 @@ void LayoutAnimationsManager::configureAnimation(
   if (type == SHARED_ELEMENT_TRANSITION) {
     sharedTransitionGroups_[sharedTransitionTag].push_back(tag);
     viewTagToSharedTag_[tag] = sharedTransitionTag;
+  } else if (type == SHARED_ELEMENT_TRANSITION_PROGRESS) {
+    if (sharedTransitionTag == "secondary") {
+      sharedTransitioinProgressAnimationsLowPriotity_.insert(tag);
```

Comment:
```suggestion
      sharedTransitionProgressAnimationsLowPriotity_.insert(tag);
```

Reply:
sharedTransitionProgressAnimationsLowPriority

there's a typo in `priority` as well 🙊